README.md

```diff
@@ -1,23 +1,43 @@
-# gradle-static-analysis-plugin
-[![](https://ci.novoda.com/buildStatus/icon?job=gradle-static-analysis-plugin)](https://ci.novoda.com/job/gradle-static-analysis-plugin/lastSuccessfulBuild/console) [![](https://img.shields.io/badge/License-Apache%202.0-lightgrey.svg)](LICENSE.txt) [![Bintray](https://api.bintray.com/packages/novoda/maven/gradle-static-analysis-plugin/images/download.svg) ](https://bintray.com/novoda/maven/gradle-static-analysis-plugin/_latestVersion)
+# Gradle static analysis plugin
```

Comment:
What about having a table of contents like section to have links to advanced configuration etc? Because otherwise, I'm afraid people will not see the inlined links.

Reply:
That wouldn't be a TOC of this document... besides there's extremely little content right now, I don't think it's easy to miss the links